.clang-format

```diff
@@ -11,14 +11,14 @@
 
 ---
 BasedOnStyle: LLVM
-AlignConsecutiveMacros: AcrossComments
-AttributeMacros:
-  - __aligned
-  - __deprecated
-  - __packed
-  - __printf_like
-  - __syscall
-  - __subsystem
+#AlignConsecutiveMacros: AcrossComments
+#AttributeMacros:
+#  - __aligned
```

Comment:
please add a comment mentioning which clang-format version started supporting this (12 in this case), see L71

Reply:
I'd rather remove all commented out lines. Keeping them in doesn't provide much.

Reply:
I think they're useful to track new features relevant to the Zephyr coding guidelines (previous file already did that)